fvtest/tril/tril/jitbuilder_compiler.cpp

```diff
@@ -51,7 +55,7 @@ int32_t Tril::JitBuilderCompiler::compileWithVerifier(TR::IlVerifier* verifier)
     auto argTypes = methodInfo.getArgTypes();
     auto argIlTypes = std::vector<TR::IlType*>{argTypes.size()};
     std::transform(argTypes.cbegin(), argTypes.cend(), argIlTypes.begin(),
-                   [&types](TR::DataTypes d){ return types.PrimitiveType(d); } );
+                   std::bind1st(std::ptr_fun(&getPrimitiveType),types));
```

Comment:
(We discussed this in person) 

Given the simplicity of `std::transform`, IMO, rather than chasing the 'functional purity', it'd be better to just do the loop, which removes the need to have a function defined, and eliminates both `std::bind1st` and `std::ptr_fun`.